src/directives/sortable/sortable.spec.ts

```diff
@@ -367,10 +384,9 @@ describe('sortable', () => {
             sortable.find('li').trigger('dragover', getEventDummy());
             sortable.find('li').trigger('drop', getEventDummy());
 
-            const event: any = sortable.emitted(MSortableEventNames.OnAdd)[0][0];
-            expect(event.sortInfo).toEqual({ canDrop: true, data: dragData, action: userDefinedAction, oldPosition: -1, newPosition: 1, grouping: userDefinedGrouping });
-            expect(sortable.emitted(MSortableEventNames.OnMove)).toBeUndefined();
-            expect(sortable.emitted(MSortableEventNames.OnRemove)).toBeUndefined();
+            expect(onAddSortInfo).toEqual({ canDrop: true, data: dragData, action: userDefinedAction, oldPosition: -1, newPosition: 1, grouping: userDefinedGrouping });
+            expect(onMoveSortInfo).toBeUndefined();
+            expect(onRemoveSortInfo).toBeUndefined();
         });
 
         it('it should handle drop correctly when moving an item to another index in the same sortable', () => {
```

Comment:
tu peux enlever le 'it' en debut de nom de test

Reply:
Ouep j'ai repassé dans les tests de sortable, droppable et draggable et j'ai enlevé tous les it